sklearn/metrics/scorer.py

```diff
@@ -482,6 +504,18 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
     return cls(score_func, sign, kwargs)
 
 
+def valid_scorers():
```

Comment:
This is introducing a public function which requires more attention from the core devs, especially since the consensus in #10712 was not to introduce such a function. I think it would be better if we kept SCORERS as is, or remove 'brier_score_loss' from it and look up deprecated scorers differently in `get_scorer`.

Also, the naming is not so great. I'd rather it start with a verb like `list_`.